zipkin-anormdb/build.gradle

```diff
@@ -0,0 +1,20 @@
+// Database drivers
```

Comment:
maybe explain this a bit more in the comment. Ex. what's expecting a property of this naming convention? when is it loaded (during tests?) how is it correlated to the anormDriverDependencies below (if at all)?